src/portal/src/i18n/lang/fr-fr-lang.json

```
@@ -128,8 +128,6 @@
         "CLI_PASSWORD_TIP": "Vous pouvez utiliser ce secret comme mot de passe quand vous utilisez la CLI de docker/helm pour accéder à harbor.",
```

Keep same case everywhere 
```suggestion
        "CLI_PASSWORD_TIP": "Vous pouvez utiliser ce secret comme mot de passe quand vous utilisez la CLI de docker/helm pour accéder à Harbor.",
```